packages/c/chessboard-js.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "chessboard-js",
   "description": "JavaScript chessboard widget",
-  "keywords": [],
+  "keywords": [
+    "chess",
+    "chessboard",
+    "jquery"
+  ],
   "license": "MIT",
   "homepage": "https://chessboardjs.com",
   "repository": {
     "type": "git",
     "url": "git://github.com/oakmac/chessboardjs.git"
   },
+  "filename": "chessboard-1.0.0.min.js",
```

Comment:
```suggestion
```
As above, setting a version-specific default file will be misleading as soon as a new version is pulled in -- there is no default file we can suggest here.